serverless/main.mk

```diff
@@ -31,8 +28,9 @@ ifeq (, $(SLS))
 	$(error "aws cli toolchain is not installed or incorrectly configured.")
 endif
 
-CMD_SLS_SERVICE_INSTALL = $(NPM) install --save-dev
-CMD_SLS_SERVICE_DEPLOY = $(SLS) deploy --env $(ENV) --region $(AWS_REGION) --profile $(AWS_PROFILE) --service $(SVC) -c $(SLS_FILE)
-CMD_SLS_SERVICE_BUILD = cd $(PROJECT_PATH) && make
-CMD_SLS_SERVICE_DESTROY = $(SLS) remove --env $(ENV) --region $(AWS_REGION) --profile $(AWS_PROFILE) --service $(SVC) -c $(SLS_FILE)
-CMD_SLS_SERVICE_SECRETS_PUSH = $(CMD_SLS_SERVICE_SECRETS_PUSH)
+# Backwards Compatibility, should be removed in 2.0
+########################################################################################################################
```

Comment:
We talked about using this as a way to add compatibility in case you wanted to use `CMD_SLS_SERVICE_PLUGIN`, so these backwards compatibility features are not necessary.

Reply:
Also, what is the reason that we have to cd into the directory? It should work fine with `-c` parameter, if I'm not mistaken

Reply:
fixed